shared/actions/profile/proofs.js

```diff
@@ -263,12 +263,16 @@ function * _cancelAddProof (): SagaGenerator<any, any> {
   yield put(navigateUp())
 }
 
-function * _submitCryptoAddress (): SagaGenerator<any, any> {
+function * _submitCryptoAddress (action: SubmitBTCAddress | SubmitZcashAddress): SagaGenerator<any, any> {
   yield put(_cleanupUsername())
   const address = yield select(state => state.profile.username)
```

Comment:
🐑 I know this wasn't you, but would be nice to update this to the typed selector (optional)